www/js/app.js

```diff
@@ -1425,9 +1430,9 @@
             );
         }
 
-        articleListHeaderMessageDiv.html(message);
+        articleListHeaderMessageDiv.innerHTML = message;
```

Comment:
@Jaifroid , do you want me to do remove all usage of `.innerHTML` ? Or should we do that in a different PR or do we even need to do it?

Reply:
@dheerajdlalwani  No, we have #824 for that, and in some cases we have to keep it. In this case it could be replaced, but it should be a different PR.